capstone/src/main/webapp/youtubeScript.js

```diff
@@ -314,7 +314,7 @@ function perspectiveToxicityScale(attributeAverages) {
   const knoopScore = averageToxicityScore * 7000;
   let knoopLow;
   let knoopHigh;
-  let mohs;
+  let mohsScore;
```

Comment:
Is it possible that we can delete this youtubeScript.js file (and add it to .gitignore), then have the TypeScript compiler generate the file?  That way we don't have 2 copies of the same code checked in, and if anyone wants to update the logic they only need to modify the .ts file.

Possibly we'll want to create a script like `compile.sh` which runs the TypeScript compiler (and if needed, copies the output files to some specific places).  We can also have scripts like `deploy_dev.sh` and `deploy_appengine.sh` which do the compile commands, and then the mvn commands - this way we are always sure that appengine is running the latest .ts code

Reply:
Just to follow up, Olu and I discussed using scripts inside of a package.json file, so users can just run commands like `npm install` and then custom scripts like `npm run deploy_dev`, `npm run deploy_appengine`, etc.  That should be more standard and result in less files than if we used .sh scripts

Reply:
I have done research of my own and I agree that scripts inside the package.json file are a great idea. In "add package.json" I have added a packages.json file. This will add support for the inbuilt `npm install` script as well as custom scripts I have added. These include 'npm run installTS' which initiates TypeScript installation and 'npm run deploy_appengine' which deploys app engine.